src/platform/packages/shared/kbn-es-query/src/utils.ts

```diff
@@ -52,6 +52,11 @@ export function isCCSRemoteIndexName(indexExpression: string): boolean {
     // Thus, whatever it is, this is definitely not a remote index.
     return false;
   }
+
+  const idx = indexExpression.indexOf(':');
+  // Check to make sure the remote cluster separator ':' isn't actually a selector separator '::'
+  const isSelector = indexExpression.startsWith('::', idx);
```

Comment:
Wow, I never knew you could pass an index to `startsWith` 😂

Reply:
ha, me neither :)